velox/functions/sparksql/String.h

```diff
@@ -283,6 +283,82 @@ struct EndsWithFunction {
   }
 };
 
+/// substring_index function
```

Comment:
This statement is redundant. Let's remove.

Reply:
Removed.